From 9c06a5ce5fa788cbbe3caecc0ffbc1440849a5cc Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sun, 1 Aug 2021 18:35:17 +0200
Subject: [PATCH 109/N] configure.ac: don't check for clock_ functions

They shouldn't be exposed on Windows and lead to winpthread being linked in

Signed-off-by: Naveen M K <naveen521kk@gmail.com>
---
 configure.ac | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configure.ac b/configure.ac
index 3d31cf3..6afc9c9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4394,6 +4394,9 @@ char *r = crypt_r("", "", &d);
     [])
 )
 
+case $host in
+  *-*-mingw*) ;;
+  *)
 AC_CHECK_FUNCS(clock_gettime, [], [
     AC_CHECK_LIB(rt, clock_gettime, [
         LIBS="$LIBS -lrt"
@@ -4414,6 +4417,8 @@ AC_CHECK_FUNCS(clock_settime, [], [
         AC_DEFINE(HAVE_CLOCK_SETTIME, 1)
     ])
 ])
+  ;;
+esac
 
 AC_MSG_CHECKING(for major, minor, and makedev)
 AC_LINK_IFELSE([AC_LANG_PROGRAM([[
-- 
2.35.1

